db/re/item_db.txt

```diff
@@ -4030,7 +4030,14 @@
 5893,RCC2013_2ND_CROWN,RCC2013 2ND CROWN,4,20,,2500,,7,,1,0xFFFFFFFF,63,2,256,,1,0,1107,{ bonus bAllStats,4; skill "AL_TELEPORT",1; },{ sc_start SC_SPEEDUP0,-1,25; },{ sc_end SC_SPEEDUP0; }
 5894,RCC2013_3RD_CROWN,RCC2013 3RD CROWN,4,20,,2500,,7,,1,0xFFFFFFFF,63,2,256,,1,0,1108,{ bonus bAllStats,3; skill "AL_TELEPORT",1; },{ sc_start SC_SPEEDUP0,-1,25; },{ sc_end SC_SPEEDUP0; }
 5895,RCC2013_ANV_Hat,RCC2013 ANV Hat,4,20,,2500,,7,,1,0xFFFFFFFF,63,2,256,,1,0,1109,{ bonus bAllStats,1; bonus2 bAddClass,Class_All,2; bonus bMatkRate,2; },{},{}
+5900,Divine_Guard_Hat,Divine Guard Hat,4,0,,200,,10,,1,0x00004000,63,2,256,,100,1,1121,{ .@r = getrefine(); bonus2 bSubSize,Size_All,5; if(.@r>=7) { bonus3 bAutoSpell,"AL_HEAL",max(getskilllv("AL_HEAL"),5),10; } if(.@r>=9) { bonus2 bSkillAtk,"LG_RAYOFGENESIS",20; } },{},{}
+5901,Focus_Beret,Focus Beret,4,0,,200,,10,,1,0x00008000,63,2,256,,100,1,1122,{ .@r = getrefine(); bonus4 bAutoSpellOnSkill,"SR_DRAGONCOMBO","SR_FALLENEMPIRE",1,10; if(.@r>=7) { autobonus "{ bonus bAspd,2; }",30,5000,0,"{ specialeffect2 EF_ENHANCE; }"; } if(.@r>=9) { bonus2 bSkillAtk,"SR_FALLENEMPIRE",30; } },{},{}
+5902,Harvester_Hat,Harvester Hat,4,0,,200,,10,,1,0x00040000,63,2,256,,100,1,1123,{ .@r = getrefine(); bonus2 bAddMonsterDropItem,6211,100; bonus2 bAddMonsterDropItem,6212,100; if(.@r>=7) { bonus2 bAddMonsterDropItem,6210,50; } if(.@r>=9) { bonus2 bSkillAtk,"GN_CARTCANNON",15; } },{},{}
+5903,Deadman_Bandana,Deadman Bandana,4,0,,200,,15,,1,0x00810204,63,2,256,,99,1,1124,{ .@r = getrefine(); bonus bCritical,.@r*2; bonus bFixedCastrate,-2; bonus3 bAutoSpell,"SL_SWOO",max(getskilllv("SL_SWOO")),10; if(.@r>=7) { bonus bMatk,20; bonus2 bCastrate,"SL_SWOO",3; } if(.@r>=9) { bonus bVariableCastrate,-20; bonus2 bSkillUseSP,"SL_STUN",5; bonus2 bSkillUseSP,"SL_SMA",5; } },{},{}
```

Comment:
https://www.divine-pride.net/database/item/5903
`This item cannot be traded with other accounts. ` -> update item_trade.txt
`When being physically attacked` -> bAutoSpellWhenHit instead of bAutoSpell.

`If upgraded to +7 or higher,, MATK + 20, Increase casting rate of Eswoo.(3%) ` I think it increases the chance of the autocast, not the casting time. For me it should be
```
.@rate = 1;
if (.@r>=7) { bonus bMatk,20; .@rate += 3; }
bonus3 bAutoSpellWhenHit,"SL_SWOO",getskilllv("SL_SWOO"),(.@rate*10);
``